src/python_routing_v02/troute/routing/compute.py

```diff
@@ -899,7 +900,18 @@ def compute_diffusive_routing(
                     trib_flow = np.append(trib_flow, i[1][x, ::3], axis = 0)  
                     
         # create DataFrame of junction inflow data            
-        junction_inflows = pd.DataFrame(data = trib_flow, index = trib_segs)
+        junction_inflows = pd.DataFrame(data = trib_flow, index = trib_segs)        
+     
+        # very very temporary installment for natual channel x-section data
+        inland_bathyNC = nc.Dataset('/glade/work/donghakim/cape_fear/bathy/eHydro_ned_cross_sections_2021_12_29.nc')
+        # - The current eHydro data doesn't cover Cape fear river, so for pure testing purpose we borrow 
+        #   three x-sec bathy data from arbitarily picked three segments (they are connected in series)
+        comid_bathy = np.unique(inland_bathyNC['comid'][:])[1:4]
```

Comment:
this sort of stuff is too bespoke for a particular test and should not be committed into master. Instead, think through how you would design the workflow so that this sort of decision/input could be managed way higher up in the workflow.